ffi/src/lib.rs

```diff
@@ -553,6 +591,19 @@ fn get_default_engine_impl(
     Ok(engine.into())
 }
 
+#[cfg(feature = "sync-engine")]
+fn get_sync_engine_impl(
+    allocate_error: AllocateErrorFn,
+) -> DeltaResult<Handle<SharedExternEngine>> {
+    use delta_kernel::engine::sync::SyncEngine;
+    let engine = SyncEngine::new();
```

Comment:
probably simpler to just use a fully qualified path?
```suggestion
    let engine = delta_kernel::engine::sync::SyncEngine::new();
```